pygmt/tests/test_meca.py

```diff
@@ -15,12 +15,10 @@
 @pytest.mark.mpl_image_compare
 def test_meca_spec_dictionary():
     """
-    Test supplying a dictionary containing a single focal mechanism to the
-    `spec` argument.
+    Test supplying a dictionary containing a single focal mechanism to the spec
+    argument.
```

Comment:
```suggestion
    parameter.
```
I guess we should also use parameter here.